waspc/src/Wasp/Generator/Common.hs

```diff
@@ -57,6 +58,9 @@ prismaVersion :: SV.Version
 --       Then, make sure `data/Generator/templates/sdk/wasp/prisma-runtime-library.d.ts` is up to date.
 prismaVersion = SV.Version 5 19 1
 
+reactRouterVersion :: SV.ComparatorSet
+reactRouterVersion = SV.backwardsCompatibleWith $ SV.Version 5 3 3
```

Comment:
A single place to define the version which we use in the validation code.

Reply:
This is client specific dep though, right? So it should not be here but some lower level, under client Generator stuff? Or does SDK complicate this? But then maybe it should go under SDK? Or maybe it is ok here? OK you will know better, but wanted to check with you.

Reply:
Ok, that makes sense to me! Resolve this comment when happy.